amqprs/src/api/tls.rs

```diff
@@ -64,17 +69,16 @@ impl TlsAdaptor {
     /// Panics if private key is invalid.
     pub fn with_client_auth(
```

Comment:
Had to make a breaking change here due to PrivateKeyDer and CertificateDer now accepting a lifetime param